task1_string_manipulation/helper.py

```diff
@@ -0,0 +1,52 @@
+
+def removeMultyOccurence(str):
+    temp = ""
+
+    for s in str:
+        if s.lower() not in temp.lower():
+            temp = temp + s
+
+    return temp
+
```

Comment:
Avoid using variable names like temp, and s. Use a more descriptive variable name.